src/components/EmptyState/EmptyState.module.css

```diff
@@ -0,0 +1,46 @@
+.emptyState_Container {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  width: 344px;
+  height: 328px;
+}
+
+.icon_container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
use util class here

Reply:
Thank you for pointing this out, have corrected it in the latest commit.